docs/release-notes/8.16.asciidoc

```diff
@@ -0,0 +1,155 @@
+
+[discrete]
+[[features-8.16.0]]
+==== New features
+
+Introduces the entity store as a technical preview feature, which allows observed, imported, integrated, or uploaded entities to be stored persistently ({kibana-pull}192806[#192806]).
+* Introduces Knowledge Base for Elastic AI Assistant, which allows you to specify information for AI Assistant to remember when responding to your queries ({kibana-pull}186566[#186566], {kibana-pull}192665[#192665]).
+* Enables detection rules to automatically execute system actions such as opening a case ({kibana-pull}183937[#183937]). 
+* Adds syntax validation for {esql} queries ({kibana-pull}189780[#189780]).
+* Provides a way to view {es} queries that run during rule execution. This option is provided for {esql} and EQL rules only ({kibana-pull}191107[#191107]).
+* Allows you to create and update a rule even when some data-related validation errors are present in the query field ({kibana-pull}191487[#191487]).
+* Introduces a new advanced setting `securitySolution:enableVisualizationsInFlyout`. When enabled, you can examine alerts and events in the **Visualize** tab, which provides a more detailed view of the event analyzer and Session View ({kibana-pull}194012[#194012], {kibana-pull}192531[#192531], {kibana-pull}192643[#192643]).
+* Creates a new advanced setting `securitySolution:excludedDataTiersForRuleExecution` that allows you to exclude cold and frozen data from rule execution. This setting does not apply to {ml} rules ({kibana-pull}186908[#186908]). 
+* Introduces the ability to add notes to alerts and events, in addition to Timeline ({kibana-pull}186787[#186787], {kibana-pull}186807[#186807], {kibana-pull}186931[#186931], {kibana-pull}186946[#186946], {kibana-pull}187214[#187214], {kibana-pull}193373[#193373]).
+//PR for notes feature is incoming.
+* Enhances the Insights in the section in the alert and event details flyouts by providing available misconfiguration and vulnerabilities findings ({kibana-pull}195509[#195509]).
+* Removes the `securitySolution:enableAssetCriticality` advanced setting and enables <<asset-criticality, asset criticality>> workflows by default ({kibana-pull}196270[#196270])
+* Adds RBAC for Elastic AI Assistant's knowledge base ({kibana-pull}195733[#195733]).
+* Adds RBAC for Attack Discovery ({kibana-pull}188788[#188788]).
+* Adds historical results to the Data Quality dashboard and updates its UI ({kibana-pull}191898[#191898], {kibana-pull}196127[#196127]). 
+* Enables agentless deployment for Elastic's Cloud Security Posture Management integration ({kibana-pull}191557[#191557]).
```

Comment:
Please move this up, its a new capability. And reword- Enables agentless deployment for Elastic’s Cloud Security Posture Management integration and new Cloud Asset Inventory integration

Reply:
@benironside we are also introducing the new Cloud asset inventory integration. This is not yet mentioned. Is it because this feature is in Tech preview? cc @tinnytintin10

Reply:
Moved this summary up as requested in [ca9d5ef](https://github.com/elastic/security-docs/pull/5977/commits/ca9d5ef269a47f7fc0c7cff0a0d1080c9e54ad67).

Reply:
Resolved in [779327c](https://github.com/elastic/security-docs/pull/5977/commits/779327cf3dab890ed70dccfebc26b8d80e0758fc) and [ff326f6](https://github.com/elastic/security-docs/pull/5977/commits/ff326f6267d7cdc99e78ac4f0963cb73f9719641).